### ISSUE TITLE
"Something went wrong" - an error during publishing a bot to Azure

### ISSUE THREAD

**Issue by balyetcait** [microsoft/BotFramework-Composer#6648]
Hi!
At the begining the publishing system worked fine. 
The next day and onwards it started to refuse to work  with the following error 
![image](https://user-images.githubusercontent.com/81765903/113319314-f93e3080-9319-11eb-86da-6be6b3f5a47b.png)

Tried to figure out what happened and found somewhat of the description for the error in the profile edit
![image](https://user-images.githubusercontent.com/81765903/113319774-79fd2c80-931a-11eb-919e-ffbee0e54ad7.png)

It didn't make it much crearer though.

I'm using 
subscription "Azure for Students"
Bot Framework Composer v1.4.0
Chrome
Windows 10 x64

**Comment by v-kydela** [comment]
Thank you for the report, @balyetcait 

1. Can you provide numbered steps to reproduce the problem?
2. When you say "At the begining the publishing system worked fine" do you mean it only worked the first time you tried it or did it work multiple times?
3. When you say "Chrome" do you mean you're running Composer in a browser instead of the Electron app?

**Comment by balyetcait** [comment]
Hi
1.The issue "Something went wrong" occurs when i press okay  at the menu
![image](https://user-images.githubusercontent.com/81765903/113411916-ae76f400-93bf-11eb-8c5b-145978737f3a.png)
The detailed description is located at profile manager in the recourse selection tab after authorization
![image](https://user-images.githubusercontent.com/81765903/113412098-13324e80-93c0-11eb-828a-e778cc96ce7d.png)
![image](https://user-images.githubusercontent.com/81765903/113412132-2d6c2c80-93c0-11eb-8b2f-36717acff04b.png)
![image](https://user-images.githubusercontent.com/81765903/113412145-33faa400-93c0-11eb-9d68-a79a0c01f380.png)

2.As of matter of fact it did work only once. Though i tried to publish againg only on the next day. At that point my account was blocked because i tried to log in from another place. I thought that was the issue, but even after confirmation from mail and unblocking account the problem was still there.   
3. I use Composer app. Listed Chrome for using Azure just in case.

**Comment by v-kydela** [comment]
Thank you, @balyetcait 

In order to try and reproduce your problem, I need more detailed repro steps. Please try creating a new publish profile and recording the steps you take carefully. For example, do you select all of the optional resources like App Insights?

If I can't reproduce the problem after getting detailed repro steps, I can try to help you use the "Import existing Azure resources" option.

**Comment by balyetcait** [comment]
Tried to create a new one, same issue. As long as i'm authorized, after i get past choosing Publishing target screen i get the resource screen with the error.
Tried to sign out and relogin but the error returns as soon I’m in

![image](https://user-images.githubusercontent.com/81765903/113458615-86b57980-941b-11eb-86a9-77a7990d6ba8.png)
![image](https://user-images.githubusercontent.com/81765903/113458946-badd6a00-941c-11eb-95f2-2286d33e763c.png)
![image](https://user-images.githubusercontent.com/81765903/113458662-afd60a00-941b-11eb-9c19-313b517e9657.png)

**Comment by v-kydela** [comment]
@balyetcait - Okay, it looks like you're saying you can't even get to the part where you choose your resources because Composer is having trouble recognizing your credentials. I don't think your publish configuration will make a difference if this is a problem with your Azure account.

@cwhitten - Who should we talk to about resolving errors from AAD?

**Comment by a-b-r-o-w-n** [comment]
@balyetcait Can you try updating to the latest nightly build of composer? Go to the application settings page and check the early adopters toggle (gear icon on lower left side). Then in menu bar go to Help > Check for updates.

Once you have installed the latest, try this again. We have recently made changes to authentication when creating a publish profile which may have resolved your issue.

**Comment by v-kydela** [comment]
@balyetcait - Are you still working on this?

**Comment by balyetcait** [comment]
Hi, @v-kydela, @a-b-r-o-w-n!
Yeah, put the project on hold and missed your messages, sorry.
Updated to the latest build.

Works like a charm, thank you very much!
